### PR TITLE
Crate

### PR DESCRIPTION
**Pull request by EPashkin**
cc @GuillaumeGomez

### CODE REVIEW

**Comment by EPashkin** [comment]
@GuillaumeGomez tests passed, can you publish this version, so sourceview PR will pass CI.

**Comment by GuillaumeGomez** [comment]
Sure, thanks!

**Comment by GuillaumeGomez** [comment]
Published.

**Comment by EPashkin** [comment]
@GuillaumeGomez Tag it please.

**Comment by GuillaumeGomez** [comment]
Damn, not used to do it by hand. I'll need to update the website as well.

**Comment by GuillaumeGomez** [comment]
Tagged as well.

**Comment by EPashkin** [comment]
Thanks

File: src/types.rs [hunk 1]
```diff
@@ -187,10 +187,6 @@ impl StaticType for Vec<String> {
     }
 }
 
-pub trait InstanceType {
```
Comment (sdroege) [inline src/types.rs]: This breaks API in theory, depends on if you want to ignore that. Nobody probably uses this

Comment (GuillaumeGomez) [reply src/types.rs]: We'll keep this under silence if you don't mind. I want to wait a bit more for a bigger release. ;)

Comment (sdroege) [reply src/types.rs]: Fine with me, but it should be a conscious decision and not an accident

Comment (GuillaumeGomez) [reply src/types.rs]: I have to admit it completely slipped out of my review. Should have been more careful...

**Pull request merged**